### PR TITLE
Added use collector to the plugin list

### PR DESCRIPTION
**Pull request by sandlbn**
Summary of changes:
- Added use collector to the plugin list

Testing done:
- Manual test

@intelsdi-x/snap-maintainers

### CODE REVIEW

File: docs/plugins.yml [hunk 1]
```diff
@@ -68,6 +68,7 @@
 - intelsdi-x/snap-plugin-collector-snmp
 - intelsdi-x/snap-plugin-collector-smart
 - intelsdi-x/snap-plugin-collector-swap
+- intelsdi-x/snap-plugin-collector-use
```
Comment (nanliu) [inline docs/plugins.yml]: Is the plugin name an acronym for something? it doesn't seem really clear what it's collecting.

**Comment by nanliu** [comment]
Since it's an internal plugin, we have some housekeeping to do on the plugin to get it to a released state (sync repo, github release, etc) before merging this. This will be discussed and reviewed during tomorrow's sync up meeting.

**Comment by sandlbn** [comment]
@nanliu it's USE plugin which means Utilization, Saturation, Errors . Ok, thanks. It's only supported on Linux system.

**Comment by mjbrender** [comment]
Hey @nanliu - good call on the question around naming. @sandlbn has great documentation that will accompany it to clarify fit/usage.

**Comment by kjlyon** [comment]
As per this morning's sync meeting we haven't yet determined our full criteria for releasing internal plugins. For the time being I am labeling this as a work in progress.

**Comment by sandlbn** [comment]
@kjlyon what doesn't mean internal in snap plugin context ? is not internal for sure. http://www.brendangregg.com/usemethod.html

**Comment by nanliu** [comment]
@sandlbn, internal as released under 'intelsdi-x' org. In the meeting today, we were also discussing the need to run a code scan, document review on any new intelsdi-x org plugins. We don't have an official check list yet, but some of the task we should complete are:

* review repo description/url link
* pluginsync repo
* tag and release binary

I'll provide a link to the document as soon as I finish the draft.

**Comment by nanliu** [comment]
@sandlbn, the draft for plugin review is here: https://intelsdi.atlassian.net/wiki/display/SL/Snap+Plugin+Catalog

We need to complete the following tasks:
- [ ] PR needs review and merge: https://github.com/intelsdi-x/snap-plugin-collector-use/pull/3
- [ ] github tag and plugin release
- [ ] protex code scan

**Pull request merged**